ui/src/infra-model/view/formgroup/formgroup.module.scss

```diff
@@ -35,8 +35,29 @@
     }
 
     &__edit-icon {
+        display: flex;
+        align-items: center;
         padding: 0 5px;
         cursor: pointer;
         margin-right: 20px;
     }
+
+    &__textarea {
+        display: flex;
+        flex: 1;
+    }
+
+    &__textarea-content {
+        flex: 1;
+        margin-right: 10px;
+    }
+
+    &__textarea-content textarea {
+        overflow-y: hidden;
+    }
+
+    &__paragraph-with-linebreaks {
+        white-space: pre-line;
+        word-break: break-all;
```

Comment:
Jos teksti on pitkä, eikä sisällä välejä, niin tämä pakottaa tekstin hyppäämään seuraavalle riville, ettei se mene päällekkäin muiden oikealla puolella olevien elementtien kanssa.